xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -87,6 +87,8 @@
 
   @VisibleForTesting
   static final String ADS_TYPE_URL_LDS = "type.googleapis.com/envoy.api.v2.Listener";
+  private static final String ADS_TYPE_URL_LDS_V3 =
```

Comment:
If `ADS_TYPE_URL_LDS` needs to be visible for testing how come `ADS_TYPE_URL_LDS_V3` doesn't need to be if the test is also migrated to v3?

Reply:
Done.